docs/usage/csi_components.md

```diff
@@ -0,0 +1,22 @@
+# (Custom) CSI Components
+
+Some provider extensions for Gardener are using CSI components to manage persistent volumes in the shoot clusters.
+In most cases, controllers for taking volume snapshots are delivered out-of-the-box as well.
```

Comment:
I stumbled over this sentence.`in most cases` of what; and what does `out-of-the-box` mean?
Maybe better (although I am missing context):
```
Most of those gardener-deployed provider extensions include controllers for taking volume snapshots.
```

Reply:
Thanks, do you think it's better now?

Reply:
Yes